### PR TITLE
Add input component

### PR DESCRIPTION
**Pull request by Rheniery**
If applied, pull request will add the input component to the project

### Other minor changes:
N/A

### Card Link:
N/A

### Design Expected Screenshot
N/A

### Implementation Screenshot or GIF
Seleção_115

### Notes:
N/A

### CODE REVIEW

File: src/components/Input/DefaultInput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from '@/components/Input/DefaultInput/DefaultInput.css';
+
+interface Props {
```
Comment (kaiomagalhaes) [inline src/components/Input/DefaultInput/index.tsx:6]: please sort these alphabetically

File: src/index.tsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
 export { default as Button } from '@/components/Button';
 
-export * from '@/components/Grid';
\ No newline at end of file
+export * from '@/components/Grid';
+
+export { default as Input } from '@/components/Input';
```
Comment (kaiomagalhaes) [inline src/index.tsx:5]: please place this on line 2

File: src/components/Input/DefaultInput/index.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  className: string;
```
Comment (kaiomagalhaes) [inline src/components/Input/DefaultInput/index.tsx:7]: you need to receive the value as well

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: src/components/Input/SimpleInput/SimpleInput.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+@import "../../../styles/theme/index.css";
+
+:root {
```
Comment (diogoribeiro) [inline src/components/Input/SimpleInput/SimpleInput.css:3]: we also need the properties for the placeholder

File: src/components/Input/DefaultInput/DefaultInput.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.input:hover,
+.input:focus {
+  border: var(--input-border-hover, none);
+  color: var(--input-color-hover, transparent);
+}
+
+.input:focus {
+  outline-color: var(--input-outline-color-focus);
```
Comment (diogoribeiro) [inline src/components/Input/DefaultInput/DefaultInput.css:21]: we also need the:
- font-size
- color
- font-weight

**Review by diogoribeiro — APPROVED** [review]

**Pull request merged**